common/reviews/api/rush-lib.api.ts

```diff
@@ -203,8 +203,10 @@ class RushConfiguration {
   public static loadFromDefaultLocation(): RushConfiguration;
   public readonly npmCacheFolder: string;
   public readonly npmTmpFolder: string;
-  public readonly npmToolFilename: string;
-  public readonly npmToolVersion: string;
+  // (undocumented)
+  public readonly packageManager: 'pnpm' | 'npm';
+  public readonly packageManagerToolFilename: string;
+  public readonly packageManagerToolVersion: string;
```

Comment:
This is a breaking API change...  not sure how we want to handle that

Reply:
It is probably OK?

---
In reply to: [151780117](https://github.com/Microsoft/web-build-tools/pull/427#discussion_r151780117) [](ancestors = 151780117)

Reply:
I reverted this.

---
In reply to: [151780117](https://github.com/Microsoft/web-build-tools/pull/427#discussion_r151780117) [](ancestors = 151780117)